TestAssets/TestProjects/PortableTestApp/Program.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace PortableApp
+{
+    public static class Program
+    {
+        public static void Main(string[] args)
```

Comment:
fixed.

---
In reply to: [66352888](https://github.com/dotnet/core-setup/pull/97#discussion_r66352888) [](ancestors = 66352888,66346482)